x-pack/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_control.js

```diff
@@ -175,7 +179,10 @@ export class TooltipControl extends React.Component {
         y: mbLngLatPoint.y + PADDING,
       },
     ];
-    return this.props.mbMap.queryRenderedFeatures(mbBbox, { layers: mbLayerIds });
+    return this.props.mbMap.queryRenderedFeatures(mbBbox, {
+      layers: mbLayerIds,
+      filter: ['==', ['get', KBN_TOO_MANY_FEATURES_PROPERTY], false],
```

Comment:
use TOO_MANY_FEATURES_FILTER from mb_filter_expressions instead of redefining the filter here